pkg/git/gitlab/gitlab_helper_test.go

```diff
@@ -56,3 +56,123 @@ func TestGetProjectPathFromRepoUrl(t *testing.T) {
 		}
 	}
 }
+
+func TestGetHostFromRepoUrl(t *testing.T) {
+	var params = []struct {
+		in  string
+		out string
+		err error
+	}{
+		{
+			"https://gitlab.cee.redhat.com/rhtap/tenants-config.git",
```

Comment:
AFAIK we should generally avoid mentioning internal URLs in upstream code, could you replace the redhat.com urls with something else?

Reply:
done